components/People/Members/index.tsx

```diff
@@ -42,12 +43,31 @@ const MembersWrapper = styled.div`
 
 const Members = () => (
   <Box>
+    <SectionTitle
+      title="와플스튜디오 운영팀"
+      subTitle="와플스튜디오 22-23 운영팀입니다."
+    />
+    <MembersWrapper>
+      {members.filter((m)=>{return m['op']}).map((member) => (
+        <Member
+          name={member.name}
+          githubId={member.githubId}
+          position={member.position}
+          introduction={member.introduction}
+          github={member.githubId}
+          facebook={member.facebook}
+          web={member.web}
+          instagram={member.instagram}
+        />
+      ))}
+    </MembersWrapper>
+    <SectionDivider></SectionDivider>
     <SectionTitle
       title="와플스튜디오 구성원"
       subTitle="와플스튜디오의 전 / 현 구성원 입니다."
     />
     <MembersWrapper>
-      {members.map((member) => (
+      {members.filter((m)=>{return !m['op']}).map((member) => (
```

Comment:
화살표 함수 보통은 이렇게 쓰긴 합니다
```suggestion
      {members.filter((m)=> !m['op']).map((member) => (
```

Reply:
prettier 적용하고 arrow function convention 적용했슴다
리뷰 감사감사합니당